manifests/04-deployment.yaml

```diff
@@ -10,6 +10,7 @@ metadata:
     exclude.release.openshift.io/internal-openshift-hosted: "true"
     include.release.openshift.io/self-managed-high-availability: "true"
     include.release.openshift.io/single-node-developer: "true"
+    kubectl.kubernetes.io/default-container: machine-approver-controller
```

Comment:
[Docs for this annotation][1] :+1:

[1]: https://kubernetes.io/docs/reference/labels-annotations-taints/#kubectl-kubernetes-io-default-container